go.mod

```diff
@@ -4,6 +4,7 @@ go 1.21.4
 
 require (
 	cloud.google.com/go/compute v1.23.3
+	cloud.google.com/go/resourcemanager v1.9.4
```

Comment:
As of now, we only used google.golang.org/api clients. Not cloud.google.com client. 
The reason is that cloud.google.com doesn't have all the APIs we need. 
I think it is very risky to use both set of clients as they might have different versioning and different dependencies  and might cause weird errors/bugs. Any thoughts?

Reply:
Actually, Anish had some reservations about using that api, but he followed up and was told that as I mentioned, We should use google.golang.org/api if we want to call ServiceNetworking APIs and even though it is in maintenance mode, it is not going to go away anytime soon. So the conclusion was to keep using google.golang.org/api. Do you remember who told you to use the other API?